extensions/wikia/SitemapXml/SpecialSitemap_body.php

```diff
@@ -43,9 +43,6 @@ public function execute( $subpage ) {
 
 		if ( ( $showIndex && !$forceOldSitemap ) || $forceNewSitemap ) {
 			$this->getOutput()->disable();
-			if ( !Hooks::run( 'SitemapPageBeforeOutput', [ $this->getRequest(), $this->getUser() ] ) ) {
```

Comment:
Good point. I'll have to think about the downgrades in `onBeforeInitialize`. Because if that part of the code or WFL decides to redirect, this execute method won't be called.

Reply:
discussed on Slack, it should be fine now as `onBeforeInitialize` prevents the sitemap code from ignoring the Output page redirect.